Add Q1 web OKR around onboarding improvements (#660)
sourcegraph/about: company/okrs/2020_q1.md

   1. Search daily, weekly, monthly active users are tracked in [pings](https://docs.sourcegraph.com/admin/pings).
   1. Search filter usage is tracked in [pings](https://docs.sourcegraph.com/admin/pings).
   1. Search mode usage is tracked in [pings](https://docs.sourcegraph.com/admin/pings).

> 1. **Identify the pain points in our onboarding**

Author: ```suggestion
1. **Identify and start addressing pain points in our onboarding**
```

>    1. Run N user tests to gather feedback on our user onboarding

Author: @christinaforney @imtommyroberts we discussed in the last web sync wording this OKR around understanding the pain points in our onboarding, so that we can follow up with a Q2 OKR aiming to solve these pain points. Do you have feedback on KRs that would be helpful to track beyond user tests (+ does this wording look good? Does it need to be more detailed?).

We also mentioned that we might shoot for a "quick win" improvement to our onboarding in the short term, while we work on getting the big picture. Do we have a sense of what that might look like, and would we want to track it in a KR here (if so, we might have to reword the 'O').

Reviewer: Wording looks good to me. 5 user tests would be a good goal before the end of Q1. We should also mention that I'd like to set up a brainstorming doc to be used asynchronously. 

Regarding the 3.15 update, the figma file is here: https://www.figma.com/file/x3d9uhuknTyhKASbqwxMVg/Onboarding-RFC-121-125?node-id=38%3A0 (please comment in the file if you have anything to add)

Not sure how to word it, perhaps something like 'Improvements to the current onboarding UI'

>    1. Run N user tests to gather feedback on our user onboarding

Author: ```suggestion
   1. Run 5 user tests to gather feedback on our user onboarding
```

>    1. Run 5 user tests to gather feedback on our user onboarding

Author: ```suggestion
   1. Run 5 user tests to gather feedback on our user onboarding
   1. Improve in-app documentation for site admin onboarding steps
```
